tests/test_connection.py

```diff
@@ -235,7 +235,7 @@ def test_connection_test(mock_connect, runner):
     assert "Host" in result.output
     assert "Password" not in result.output
     assert "password" not in result.output
-    mock_connect.assert_called_once_with(connection_name="full")
+    mock_connect.assert_called_with(connection_name="full")
```

Comment:
why did you changed it? was it required? did old assertion stop working?

Reply:
There are two calls instead of one because of sending telemetry. This is to be refactored as we discussed some time ago (explicit app module usage in plugins)